api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This must be run in project root, to pick up dataset_config/.
-FROM python:2
+FROM python:3
```

Comment:
When you tested locally, you only had this change and not the other changes from your fork, right?

Reply:
Correct